htdocs/main.php

```diff
@@ -96,7 +96,7 @@ function tplFromRequest($param)
     $tpl_data['userNumSites']         = count($site_arr);
     $tpl_data['user']['SitesTooltip'] = str_replace(
         ";",
-        "\n",
+        "<br/>",
```

Comment:
Because otherwise every site would not render on its own line. HTML doesn't understand `\n`